website/docs/faqs/Accounts/cloud-upgrade-instructions.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "How to upgrade a dbt Cloud account"
+id: "cloud-upgrade-instructions"
+description: "Instructions for upgrading a dbt Cloud account after the trial ends."
+---
+
+dbt Cloud offers [several plans](https://www.getdbt.com/pricing/) with different features that meet your needs. This document is for dbt Cloud users and explains how to select a plan in order to continue using dbt Cloud. 
+
+:::tip Before you begin
+- You **_must_** be part of the [Owner](/docs/collaborate/manage-access/self-service-permissions) user group to make billing changes. Users not included in this group will not see these options.
+- All amounts shown in dbt Cloud are in U.S. Dollars (USD)   
+:::
+
+
+## Select a plan 
+When your [14 day trial](https://www.getdbt.com/signup/) ends or if your subscription payment is past due , you'll need to select a plan in order to continue using your account:
```

Comment:
Is the behavior they will see exactly the same for the trial expiration and past due payment scenarios?

Reply:
yes that's right! your account is locked